package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
yea.  they're all good thoughts.  as you've taken a step back, let me as well.

nodeapi has a myriad of tests against it. the gitflow strategy has been awesome for 2.0.  is it time to go CI in 3.0, and dodge the complexities of gitflow altogether for the CI package?

on git-post-merge into features merged into master, they could auto merge down into release and develop to keep the 2.0 gitflow process running.